dali/operators/audio/nonsilence_op.h

```diff
@@ -142,6 +144,18 @@ class NonsilenceOperator : public Operator<Backend> {
     return true;
   }
 
+  bool SetupImpl(std::vector<OutputDesc> &output_desc, const workspace_t<Backend> &ws) override {
+    AcquireArgs(spec_, ws);
+    TensorShape<> scalar_shape = {};
+    auto curr_batch_size = ws.GetInputBatchSize(0);
+
+    output_desc.resize(detail::kNumOutputs);
+    for (int i = 0; i < detail::kNumOutputs; i++) {
+      output_desc[i].shape = uniform_list_shape(curr_batch_size, scalar_shape);
+      output_desc[i].type = DALI_INT32;
```

Comment:
Meh.... I know it probably was like that before, but:
1. int32 is small - only 2G samples - it might be enough for audio, but not enough for, say, data from oscilloscope for some scientific signal processing.
2. it should be configurable
3. it is different than what `shapes` returns (int64)

Reply:
Let's do in in a follow up together with the deprecation of reset interval